### ISSUE TITLE
RFC: Measure string sizes in bytes, not codepoints

### ISSUE THREAD

**Issue by jnvsor** [kint-php/kint#324]
Currently, Kint measures strings in codepoints, not bytes.

Not having the byte length is clearly a problem, since the byte length of a string is often more useful than the amount of codepoints, and you can't infer the amount of bytes from the amount of codepoints in a variable-sized encoding like the ubiquitous UTF-8.

The same string could be shown to be the same size in codepoints in eg. ASCII, UTF-8, and UTF-16, but have radically different lengths in bytes.

Given that a codepoint does not necessarily map to a single grapheme, this may be both a slight performance hit and not the most useful thing ever. In fact it may be more confusing than just showing plain bytes.

Anyway short version: Is anyone opposed to changing string size to bytes in a next major version?

**Comment by jnvsor** [comment]
Added in 23e2a6702c258c852e9f6302350800c830d52231
